### ISSUE TITLE
Some hotkeys don't work with azerty keyboards

### ISSUE THREAD

**Issue by x-a-n-a-x** [OpenRA/OpenRA#8354]
Sorry to bother you with this problem (since it should only affect a very few players, those who have an AZERTY keyboard) but some hotkeys don't seem to work at all on these keyboards, especially the . period.

http://logs.openra.net/?year=2015&month=05&day=31#17:19:19

**Comment by PunkPun** [comment]
Is this still an issue?

**Comment by xan2622** [comment]
> Is this still an issue? 

Yes it is. I have just tested with the Playtest 20221203. That "." hotkey doesn't work.

I used my mousewheel to zoom in, then I pressed this key, nothing happened.
Then I modified the "Reset Zoom" hotkey to another key, and this time, it worked.
